.github/workflows/quarkus-snapshot.yaml

```
@@ -0,0 +1,60 @@
+name: "Quarkus ecosystem CI"
+on:
+  workflow_dispatch:
+  watch:
+    types: [started]
+
+  # For this CI to work, ECOSYSTEM_CI_TOKEN needs to contain a GitHub with rights to close the Quarkus issue that the user/bot has opened,
+  # while 'ECOSYSTEM_CI_REPO_PATH' needs to be set to the corresponding path in the 'quarkusio/quarkus-ecosystem-ci' repository
+
+env:
+  ECOSYSTEM_CI_REPO: quarkusio/quarkus-ecosystem-ci
+  ECOSYSTEM_CI_REPO_FILE: context.yaml
+  JAVA_VERSION: 17
+
+  #########################
+  # Repo specific setting #
+  #########################
+
+  ECOSYSTEM_CI_REPO_PATH: quarkiverse-grpc-zero-default
```

This does not exist yet in the ecosystem CI

```suggestion
  ECOSYSTEM_CI_REPO_PATH: quarkiverse-grpc-zero
```